lib/python/vdm/server/HTTPListener.py

```diff
@@ -637,13 +758,21 @@ def post(database_id):
         if not inputs.validate():
             return jsonify(success=False, errors=inputs.errors)
 
+        arr = ["http-listener", "admin-listener", "internal-listener", "replication-listener", "zookeeper-listener", "client-listener"]
+        for servers in Global.SERVERS:
```

Comment:
servers variable here contains a single server, correct? Please name it server (singular), if that is the case.